src/frontend/src/views/GroupMembersPage/components/MembersTab/components/InviteModal/index.tsx

```diff
@@ -48,12 +51,35 @@ const InviteModal = ({ groupName, onClose, open }: Props): JSX.Element => {
     onClose();
   };
 
-  const getAddressArrayFromInputValue = () => {
-    return uniq(inputValue.trim().split(INPUT_DELIMITERS));
+  /**
+   * Gets user input and chunks into (what should be) email addresses.
+   *
+   * Original user input text is broken into chunks by delimiters. Repeated
+   * delimiters between chunks are taken as a single delimiter. Leading and
+   * trailing delimiters are dropped. If a given chunk is found multiple times
+   * in the input, only one copy of the chunk is returned in output array.
+   *
+   * Example:
+   *     user1@example.com,   user2@example.com, ,, ,,  , user3@example.com,,
+   *   user2@example.com, user4@example.com,,
+   * Would return as four emails in total, as you'd expect to read them.
+   *
+   * Arguable if it's best UX -- should double commas bring up an error? etc --
+   * but it's easiest to code and reasonably sane to just ignore that sort
+   * of stuff. Approach was approved by Design.
```

Comment:
i don't think this part of the comment is needed, the rest of the description is good though 👍

Reply:
That's fair. I guess I was thinking that the behavior is kind of surprising --`x@a.com,,,,,,  ,,   y@a.com` being valid is weird, frankly -- so I wanted to call out that the weirdness was acknowledged and accepted, not just inadvertent.